build.py

```diff
@@ -598,6 +598,16 @@ def backend_cmake_args(images, components, be, install_dir, library_paths):
     cargs.append(
         cmake_backend_enable(be, 'TRITON_ENABLE_METRICS', FLAGS.enable_metrics))
 
+    if (target_platform() == 'windows') and (not FLAGS.no_container_build):
+        print(
+            "Warning: Detected docker build is used for Windows, backend utility 'device memory tracker' will be disabled due to missing library in CUDA Windows docker image."
```

Comment:
Is windows containers build still disabled? Looks like you added CUPTI to windows image [here](https://github.com/triton-inference-server/server/pull/5642/commits/8db5267b1e28d8505acc870fc4eefc48a5b4bd13)?